File: java-codellama7B-AWQ-all/taskid-83_starts_one_ends-gen19.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // Given a positive integer n, return the count of the numbers of n-digit
    // positive integers that start or end with 1.
    public static long startsOneEnds(long n) {

        long result = 0;
        long lower = (long) Math.pow(10, n - 1);
        long upper = (long) Math.pow(10, n) - 1;
        for (int i = (int) lower; i <= upper; i++) {
            if (i % 10 == 1 || i / 10 % 10 == 1) {
                result++;
            }
        }
        return result;
    }
    public static void main(String[] args) {
    assert(startsOneEnds((1l)) == (1l));
    assert(startsOneEnds((2l)) == (18l));
    assert(startsOneEnds((3l)) == (180l));
    assert(startsOneEnds((4l)) == (1800l));
    assert(startsOneEnds((5l)) == (18000l));
    }

}

File: py-codellama13B-FP-all/taskid-83_starts_one_ends-gen19.py
def starts_one_ends(n: int) -> int:
    """
    Given a positive integer n, return the count of the numbers of n-digit
    positive integers that start or end with 1.
    """

    count = 0
    for i in range(10):
        for j in range(10):
            num = i * 10 ** (n - 1) + j
            if num % 10 == 1 or num // 10 ** (n - 1) == 1:
                count += 1
    return count
